backend/infrahub/core/query/node.py

```diff
@@ -454,7 +454,8 @@ async def query_init(self, db: InfrahubDatabase, **kwargs) -> None:
         WITH n1 as n, r1, a1 as a
         WHERE r1.status = "active"
         WITH n, r1, a
-        MATCH (a)-[:HAS_VALUE]-(av:AttributeValue)
+        MATCH (a)-[r:HAS_VALUE]-(av:AttributeValue)
+        WHERE %(branch_filter)s
```

Comment:
Without the filter, if the number of historical values is important for a given node the number of subquery could be very significant which would slow down the query